docs/rtd/development.rst

```diff
@@ -0,0 +1,19 @@
+===========
+Development
+===========
+
+
+Architecture
+============
+
+Yaramod is a C++ library with Python bindings capable of parsing, building, formatting and also modifying YARA rules; hence the main four parts of Yaramod are:
+
+* Parser of YARA rules
```

Comment:
You could at least point of which directories handle which parts and somehow link it with these wiki pages:

* https://github.com/avast/yaramod/wiki/Constructing-YARA-files
* https://github.com/avast/yaramod/wiki/Parsing-YARA-files
* https://github.com/avast/yaramod/wiki/Python-Bindings
* https://github.com/avast/yaramod/wiki/Writing-Modifying-Visitors

or obsolete some of them if there are things which are already in RTD documentation.